### DISCUSSION TITLE
Should we add a recall mechanism for jury to be replaced, even without breaching their agreement?

### DISCUSSION THREAD

**Discussion by luisivan** [nation3/court#7] [Ideas]
As pointed out by @gregfromstl:

> Final note: should we consider adding a "recall" mechanism to the court, in which the members haven't necessarily breached their contracts (shouldn't lose their locked tokens), but are showing unreasonable amounts of bias or some other reason that the majority of veNATION holders would be dissatisfied. It could trigger a new election at any time, resetting the one-year election cycle. Would obviously require an overwhelming majority of veNATION holders to vote that a recall is needed.

**Comment by xPi2** [comment]
I agree with @gregfromstl, this use case should be definitely included.
